core/block.js

```diff
@@ -591,6 +616,31 @@ Blockly.Block.prototype.setShadow = function(shadow) {
   this.isShadow_ = shadow;
 };
 
+/**
+ * Get whether this block is an insertion marker block or not.
+ * @return {boolean} True if an insertion marker.
+ */
+Blockly.Block.prototype.isInsertionMarker = function() {
+  return this.isInsertionMarker_;
+};
+
+/**
+ * Set whether this block is an insertion marker block or not.
+ * @param {boolean} insertionMarker True if an insertion marker.
+ */
+Blockly.Block.prototype.setInsertionMarker = function(insertionMarker) {
+  if (this.isInsertionMarker_ == insertionMarker) {
+    return;  // No change.
+  }
+  this.isInsertionMarker_ = insertionMarker;
+  if (this.isInsertionMarker_) {
+    // TODO: Why is this in Block.js instead of block_svg.js?
```

Comment:
Move this `TODO`/question to the the setColour method in `block.js`?

Reply:
As I said, this isn't for review yet.  I'm not done with cleanup.